static/assets/js/home.js

```diff
@@ -67,6 +67,13 @@ if (!PP) {
     });
 
     $(document).ready(function () {
```

Comment:
Is it necessary to have a `$(document).ready()` here? It should work without?

The `PP.initLogin()` function is called from the html - see [here](https://github.com/phenopolis/phenopolis/blob/master/templates/material_layout.html#L89)...

Note - `$(document).ready()` will be executed once the document is loaded.  `$(function(){...});` (what we have in the html) is a shortcut for `$(document).ready()` and does the exact same thing. On the other hand, `(function(){...})();` will be executed as soon as it is encountered in the Javascript (i.e. the `PP` JS module) - basically invoking itself. 

AS such there shouldn't be any need have another `document.ready` function.

Also, if possible move the indexOf lines into a separate function and call it from the initLogin or directly from the [html](https://github.com/phenopolis/phenopolis/blob/master/templates/material_layout.html#L89). 

I typically prefer to just define functions in the JS library files and then have a single `on.ready` function in the HTML where I call the necessary functions...